py-rse/correct.Rmd

```diff
@@ -11,7 +11,30 @@ But how can we be sure?
 The short is answer is that we can't be completely certain,
 but that as in science,
 we can test our expectations against reality to help us decide if we are sure enough.
-This chapter explores ways of doing that.
+
+We'll start with our Zipf's Law project structure as continued from the previous chapter:
```

Comment:
"as it was after" rather than "as continued from" maybe?

Reply:
Agreed, and I also checked the other chapters to ensure the language is consistent.